spec/features/admin/order_comments_spec.rb

```diff
@@ -8,10 +8,13 @@
     login_as_admin
   end
 
-  it 'adding comments' do
+  scenario 'adding comments to an order' do
```

Comment:
Why `it` -> `scenario`? Looks like the rest of the file always uses `it`.

Reply:
That was muscle memory kicking in. Since it's not worth in the scope of this change to refactor the whole specs I'm reverting to `it`.